ext/test/http/curl_http_test.cc

```diff
@@ -184,9 +185,10 @@ TEST_F(BasicCurlHttpTests, HttpResponse)
 TEST_F(BasicCurlHttpTests, SendGetRequest)
 {
   received_requests_.clear();
-  curl::SessionManager session_manager;
+  auto session_manager = http_client::HttpClientFactory::Create();
+  EXPECT_TRUE(session_manager != nullptr);
 
-  auto session = session_manager.CreateSession("127.0.0.1", HTTP_PORT);
+  auto session = session_manager->CreateSession("127.0.0.1", HTTP_PORT);
```

Comment:
Is this to avoid passing the URL detail every time to `CreateRequest` ?

Reply:
Correct.